### DISCUSSION TITLE
Any ETA for v1.67.2 release?

### DISCUSSION THREAD

**Discussion by dmiroshnikov** [grpc/grpc-java#11633] [Q&A] [answered]
Waiting fix for CVE-2024-7254...

**Comment by ejona86** [comment, marked as answer]
See https://github.com/grpc/grpc-java/issues/11602 . We don't have the protobuf upgrade backported to 1.67.x, and given 1.68(.1 :cry: ) is like a week away, it'll probably first be seen there.
